File: src/main/java/io/openfuture/openmessanger/repository/ChatParticipantRepository.java
package io.openfuture.openmessanger.repository;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import io.openfuture.openmessanger.repository.entity.ChatParticipant;

public interface ChatParticipantRepository extends JpaRepository<ChatParticipant, Integer> {
    List<ChatParticipant> findAllByChatId(Integer chat);
}

File: src/main/kotlin/io/openfuture/openmessanger/repository/ChatParticipantRepository.kt
package io.openfuture.openmessanger.repository

import io.openfuture.openmessanger.repository.entity.ChatParticipant
import org.springframework.data.jpa.repository.JpaRepository

interface ChatParticipantRepository : JpaRepository<ChatParticipant?, Int?> {
    fun findAllByChatId(chat: Int?): List<ChatParticipant?>?
}